src/app.tsx

```diff
@@ -21,6 +22,8 @@ try {
   settings = {};
 }
 
+const baseUrl = settings.baseUrl ? settings.baseUrl : "http://localhost:3000";
```

Comment:
Is this a good place to use the "null coalescing" we now have access to (TS 3.7)?

`settings.baseUrl ?? "http://localhost:300"` ??

Reply:
Yes, exactly.